README.md

```diff
@@ -464,6 +464,31 @@ response:
 }
 ```
 
+## Rates
+
+### Get Rate
+```js
+const rf = require('routefusion-sdk').Instance();
+
+const body = { 
+  source_currency: "USD",
+  destination_currency: "MXN"
+}
+
+rf.getRate(body)
+  .then(resp => resp)
+  .catch(err => err)
+```
+
+response
+
+```json
+{
+    "source_currency": "USD",
+    "destination_currency": "MXN" 
```

Comment:
Shouldn't this have `rate` or something in it?

Reply:
whoops, yarp